paddle/fluid/framework/parallel_executor.cc

```diff
@@ -319,18 +304,35 @@ ParallelExecutor::ParallelExecutor(
   }
 
   if (build_strategy.enable_parallel_graph_) {
+#ifdef PADDLE_WITH_CUDA
```

Comment:
put this block of code in ParallelSSAGraphExecutor?

Reply:
done.